dotcom-rendering/src/components/Card/Card.stories.tsx

```diff
@@ -1137,3 +1129,92 @@ export const WithLetterDesignAndShowQuotedHeadline = () => {
 
 WithLetterDesignAndShowQuotedHeadline.storyName =
 	'WithLetterDesignAndShowQuotedHeadline';
+
+export const WithSpecialPaletteVariations = () => {
+	const containerPalettes = [
+		'InvestigationPalette',
+		'LongRunningPalette',
+		'SombrePalette',
+		'BreakingPalette',
+		'EventPalette',
+		'EventAltPalette',
+		'LongRunningAltPalette',
+		'SombreAltPalette',
+		'SpecialReportAltPalette',
+	] as const satisfies readonly DCRContainerPalette[];
+
+	return (
+		<>
+			{containerPalettes.map((containerPalette) => (
+				<FrontSection
+					title={containerPalette}
+					discussionApiUrl=""
+					editionId={'UK'}
+					containerPalette={containerPalette}
+					key={containerPalette}
+				>
+					<CardWrapper>
+						<Card
+							{...basicCardProps}
+							containerPalette={containerPalette}
+							imagePositionOnDesktop="left"
+						/>
+					</CardWrapper>
+				</FrontSection>
+			))}
+		</>
+	);
+};
+
+export const DynamoWithSpecialPaletteVariations = () => {
+	const containerPalettes = [
```

Comment:
Nitpick but given `containerPalettes` appears a couple of times in this PR is it worth combining them? Or is there scope for them to get out of sync?